src/core/gpu_vk.c

```diff
@@ -1932,8 +1937,40 @@ bool gpu_init(gpu_config* config) {
       .pNext = &enableShaderDrawParameter
     };
 
+    VkPhysicalDeviceAccelerationStructureFeaturesKHR enableAccelerationStructure = { // Raytracing (will be linked in later)
+      .sType = VK_STRUCTURE_TYPE_PHYSICAL_DEVICE_ACCELERATION_STRUCTURE_FEATURES_KHR,
+      .accelerationStructure = true
+    };
+
     if (config->features) {
-      VkPhysicalDeviceShaderDrawParameterFeatures supportsShaderDrawParameter = { .sType = VK_STRUCTURE_TYPE_PHYSICAL_DEVICE_SHADER_DRAW_PARAMETER_FEATURES };
+      // Check required extensions for raytracing
```

Comment:
Multiple problems with this next section: Does not give the user the ability to disable raytracing support; goes behind Lovr's back and gets extensions directly from vkEnumerateDeviceExtensionProperties instead of integrating with the existing check-extensions system (although I think bjorn said he was removing that when he merges with dev).

Reply:
- re disabling raytracing: I think it's okay if you can't disable.  As a single data point, mesa's AMD driver totally ignores whether you enable/disable raytracing.  At first I built some complicated system to let Lua request individual features, but it turns out drivers don't really care that much what you enable (with a few exceptions like the 'robustBufferAccess' feature).